javascript-light-bulb/main.js

```diff
@@ -0,0 +1,19 @@
+/*
+- once the click occurs change the two class names.
+- once you click the button again it changes back to the original.
+*/
+var $button = document.querySelector('.button-color');
+
+var $pageColor = document.querySelector('html');
+
+function lightSwitch(event){
+  if($button.className === 'button-color light' && $pageColor.className === 'html light'){
```

Comment:
By doing it this way, the action you take in your JavaScript is dependent on checking the current state of the element, which means your JavaScript is reliant on information from another language to know what to do. Instead, you should make a variable to keep track of if the light is currently on or off, and use that variable's value to determine what action to take on the DOM